python_dependencies/backend/de/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pandas~=2.2.2
+scipy~=1.13.1
+pydantic~=1.10.7
+numba~=0.59.1
+cellxgene_ontology_guide~=1.0.0
+tiledb
```

Comment:
I think those should be pinned, or you risk to run into a version incompatibility issue if the pipeline picks up a newer version.